features/step_definitions/cost_steps.rb

```diff
@@ -0,0 +1,16 @@
+Given(/^I have ordered "(.*?)" "(.*?)"$/) do |num, item|
+  order = num.to_i.times{Order.new(item.to_s)}.to_a
+end
+
+Given(/^I have ordered "(.*?)" "(.*?)" and "(.*?)" "(.*?)"$/) do |num1, item1, num2, item2|
+  order = num1.to_i.times{Order.new(item1.to_s)}.to_a
+  order << num2.to_i.times{Order.new(item2.to_s)}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.